doc/src/install_guide/installation_guide.md

```diff
@@ -11,202 +11,149 @@
 
 The OPAE SDK has been tested on the following configurations.
 
-* Hardware: Intel(R) FPGA Programmable Acceleration Cards: Arria(R) 10 GX, D5005, and N3000.
-* Operating System: Tested on Fedora 31, with Linux kernel 5.7.
+* Hardware: Intel(R) FPGA Programmable Acceleration Cards: Arria(R) 10 GX, N3000.
+* Operating System: Tested on Fedora 32, with Linux kernel 5.8.
 * Arria&reg 10 GX FPGA FIM version: 1.0.3 (1.0 Production)
 
 ## How to download the OPAE SDK ##
 
 OPAE SDK releases are available on [GitHub](https://github.com/OPAE/opae-sdk/releases).
 Source code for the OPAE DFL device driver for Linux is also available on [GitHub](https://github.com/OPAE/linux-dfl).
 
-## Software requirements ##
+## Install the Fedora 32 ##
```

Comment:
Should this guide be specific to Fedora?

Reply:
i remember that Rita said the OPAE 2.0 is focus on Fedora 32